src/topics/lda_scvb.cpp

```diff
@@ -184,5 +185,18 @@ double lda_scvb::compute_doc_topic_probability(learn::instance_id doc,
     return (doc_topic_count_.at(doc).at(topic) + alpha_)
            / (doc_sizes_.at(doc) + num_topics_ * alpha_);
 }
+
+stats::multinomial<topic_id> lda_scvb::topic_distrbution(doc_id doc) const
+{
+    // TODO: Replace the count vectors with a multinomial rather than creating
+    // it here
+    stats::multinomial<topic_id> result = stats::multinomial<topic_id>();
+    for (topic_id tid{0}; tid < num_topics_; ++tid)
+    {
+        result.increment(tid, doc_topic_count_.at(doc).at(tid));
```

Comment:
It seems like I do need to incorporate alpha, but this isn't setting the probability (multinomial doesn't provide an interface for doing so) which is what compute_doc_topic_probability() would get me, no?